default/python/turn_events/turn_events.py

```diff
@@ -17,21 +17,22 @@ def execute_turn_events():
     # creating fields
     systems = fo.get_systems()
     radius = fo.get_universe_width() / 2.0
-    if random() < max(0.0003 * radius, 0.03):
-        if random() < 0.4:
-            field_type = "FLD_MOLECULAR_CLOUD"
-            size = 5.0
-        else:
-            field_type = "FLD_ION_STORM"
-            size = 5.0
+    field_types = {
+        1: "FLD_MOLECULAR_CLOUD",
+        2: "FLD_ION_STORM",
+        3: "FLD_NANITE_SWARM",
+        4: "FLD_METEOR_BLIZZARD",
+        5: "FLD_VOID_RIFT"
+    }
 
+    if random() < max(0.00015 * radius, 0.03):
+        field_type = field_types.get(randint(1, 5), "FLD_ERROR")
```

Comment:
Don't hardcode indices for selecting the field type. Use `field_types[0, len(field_types)]` or `random.choice(field_types)` instead, where the latter is even more comprehensible.